card-tricks/hemartin/card_tricks.go

```diff
@@ -0,0 +1,55 @@
+package cards
+
+// GetItem retrieves an item from a slice at given position. The second return value indicates whether
+// the given index exists in the slice or not.
+func GetItem(slice []int, index int) (int, bool) {
+	if index < 0 || len(slice) == 0 {
+		return 0, false
+	}
+
+	if index > len(slice)-1 {
```

Comment:
```suggestion
	if index >= len(slice) {
```
Or even better, integrating your previous check:
```
if index < 0 || index >= len(slice)
```